index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="style.css">
+        <title>Document</title>
+    </head>
+
+    <body>
+
+        <header id="header">
+            <h1>Original Trombones</h1>
+            <nav id="nav-bar">
+
+                <a class="nav-link" href="">Features</a>
+                <a class="nav-link" href="">How it Works</a>
+                <a class="nav-link" href="">Pricing</a>
+
+            </nav>
```

Comment:
Kindly, use` ul` tag to make the navigation menu
for exemple : 
`<nav id="nav-bar">`
  `<ul>`
   ` <li><a class="nav-link" href="">Features</a></li>`
`  </ul>`
`</nav>`
and remove empty line.